R/verb-copy-to.R

```diff
@@ -107,6 +107,9 @@ auto_copy.tbl_sql <- function(x, y, copy = FALSE, ...) {
 #' @param con A database connection.
 #' @param df A local data frame. The data is written directly in the SQL query
 #'   so it should be small.
+#' @param types a character vector giving variable types to use for the columns.
```

Comment:
```suggestion
#' @param types A character vector giving variable types to use for the columns.
```

Are the types automatically translated to backend specific types? That's what the link to the SQL docs imply to me. But I don't think that's how the code works?

Reply:
Ah, good point. I simply copied this from the `copy_to()` documentation but it does not really make sense to point to the SQLite documentation.